core/modules/layout/layout.module

```diff
@@ -1549,3 +1555,36 @@ function layout_set_handler_contexts(array &$contexts, array $all_contexts, arra
   // All required contexts populated successfully.
   return TRUE;
 }
+
+/**
+ * Gets the title description for the current layout.
+ *
+ * @param $layout
+ *   Layout object.
+ *
+ * @return $description
+ *   Description of title source.
+ */
+function layout_get_title_description($layout) {
```

Comment:
Let's start adding code hints for all new code. `$layout` should be `Layout $layout` indicating that only `Layout` objects should be allowed to be passed in here. It also helps with auto-complete within IDEs.